node/malus/Cargo.toml

```diff
@@ -13,7 +13,7 @@ name = "malus"
 path = "src/malus.rs"
 
 [dependencies]
-polkadot-cli = { path = "../../cli", default-features = false, features = ["cli", "malus", "polkadot-native"] }
+polkadot-cli = { path = "../../cli", features = [ "malus", "rococo-native", "kusama-native", "westend-native", "polkadot-native" ] }
```

Comment:
AFAIK, we're only using rococo-local and wococo-local in zombienet tests
cc @pepoviola please correct me if I'm wrong

Reply:
```suggestion
polkadot-cli = { path = "../../cli", features = [ "malus", "rococo-native", "polkadot-native" ] }
```

Reply:
As errors are not propagated in zombienet, I would not do this. Before switching to edition 2021 we also compiled with the exact same set of features.